lib/src/navigation/models/organizers.dart

```diff
@@ -4,16 +4,16 @@ import 'package:recase/recase.dart';
 import '../../routing/controls.dart';
 
 abstract class Organizer {
-  final String name;
-  final OrganizerType type;
   final List<Organizer> organizers;
+  final OrganizerType type;
+  final String name;
   Organizer? parent;
 
   /// Abstract class for organizer panel in the left.
   Organizer(this.name, this.type, this.organizers);
 }
 
-enum OrganizerType { folder, component, category }
+enum OrganizerType { category, folder, component }
```

Comment:
⛏️ - If this was changed around to be alpha order, `folder` needs to go at the end.